src/modules/0.0.14/aws_elb/sql/create_custom_constraints.sql

```diff
@@ -0,0 +1,14 @@
+create or replace function check_load_balancer_availability_zones(_load_balancer_name varchar, _availability_zones varchar[]) returns boolean
+language plpgsql security definer
+as $$
+declare
+  _number_of_records integer;
+begin
+  select count(*) from availability_zone where name = any(_availability_zones) into _number_of_records;
```

Comment:
oh interesting this works, i've always done select x into y from.... not the whole query and then put that in the variable.

Reply:
Well, the whole query here is a singular value because of the `count`

Reply:
Oh, I see what you mean now wrt to the syntax. The whole `into` syntax is a bit weird, though.